flytekit/annotated/condition.py

```diff
@@ -175,6 +176,7 @@ def _compute_outputs(self, n: Node) -> Union[Promise, Tuple[Promise]]:
             for x in output_var_sets[1:]:
                 curr = curr.intersection(x)
         promises = [Promise(var=x, val=NodeOutput(sdk_node=n, sdk_type=None, var=x)) for x in curr]
+        # TODO: Is there a way to add the Python interface here? Currently, it's an optional arg.
```

Comment:
also @kumare3

Reply:
there is no interface at this point, what we will need to do is create an interface dynamically here

Reply:
that anyways is the right way

Reply:
but we're in the `self._last_case` condition here, even here we don't have the interface?